wbooks/src/app/screens/Library/index.tsx

```diff
@@ -1,45 +1,24 @@
 import React, { useEffect } from 'react';
-import { FlatList, ListRenderItem, SafeAreaView } from 'react-native';
-import { StackNavigationProp } from '@react-navigation/stack';
+import { SafeAreaView } from 'react-native';
 import { useDispatch, useSelector } from 'react-redux';
-import SingleBook from '@components/SingleBook';
-import SCREENS from '@constants/screens';
-import { Book } from '@interfaces/books';
+import BookList from '@components/BookList';
 import { BookState } from '@interfaces/bookState';
 import { RootState } from '@interfaces/rootState';
-import { LibraryType } from '@interfaces/screens';
-import { LibraryStackParamList } from '@interfaces/navigatorParamLists';
 import actionCreators from '@redux/book/actions';
-import { trimLineBreaks } from '@utils/stringUtils';
 
 import styles from './styles';
 
-interface Props {
-  navigation: StackNavigationProp<LibraryStackParamList, LibraryType>;
-}
-
-function Library({ navigation }: Props) {
+function Library() {
   const dispatch = useDispatch();
   const { books } = useSelector<RootState, BookState>(state => state.books);
 
   useEffect(() => {
     dispatch(actionCreators.getBooks());
   }, [dispatch]);
 
-  const renderBooks: ListRenderItem<Book> = ({ item, index }) => (
-    <SingleBook
-      bookCover={item.imageUrl}
-      bookTitle={trimLineBreaks(item.title)}
-      bookAuthor={item.author}
-      onPressBook={() => navigation.navigate(SCREENS.BOOK_DETAILS)}
-      style={index === 0 ? styles.firstItem : {}}
-    />
-  );
-  const keyExtractor = ({ id }: Book) => String(id);
-
   return (
     <SafeAreaView style={styles.container}>
```

Comment:
The `BookList` component already have the `SafeAreaView`

Reply:
I think it might be more beneficial to remove the `SafeAreaView` from the `BookList` component rather than the `Library` screen, so any potential new features in the `Library` screen would also be wrapped inside it.